File: leetcode/easy/2496/2496.py
class Solution:
    def maximumValue(self, strs: List[str]) -> int:
        lenth : int = len(strs)
        ans : int = 0
        for i in range(lenth):
            if strs[i].isdigit():
                ans = max(ans, int(strs[i]))
            else:
                ans = max(ans, len(strs[i]))
        return ans

File: leetcode/easy/2496/2496.cpp
#include<iostream>
#include<vector>
#include<set>
#include<string>
#include<queue>
#include<cstring>
#include<cctype>
using namespace std;
class Solution 
{
public:
    int maximumValue(vector<string>& strs) 
    {
        int len = strs.size();
        int maxx = -1;
        for(int i = 0;i<len;i++)
        {
            int num = 0;
            bool flag = 0;
            for(int j = 0;j<strs[i].size();j++)
            {
                if(!isdigit(strs[i][j]))
                {
                    flag  = 1;
                    break;
                }
                num = num*10+strs[i][j]-'0';
            }
            if(flag) maxx = max(maxx,strs[i].size());
            else maxx = max(maxx,num);
        }
        return maxx;
    }
};